### PR TITLE
Explain how to dump HIR

### PR DESCRIPTION
**Pull request by theotherphil**

### CODE REVIEW

File: src/hir.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ been desugared away (as an example, `for` loops are converted into a
 
 This chapter covers the main concepts of the HIR.
 
+You can view the HIR representation of your code by passing the
+`-Zunpretty=hir-tree` flag to rustc, for example by setting the `RUSTFLAGS`
+environment variable:
+
+```
+RUSTFLAGS=-Zunpretty=hir-tree cargo build
+```
```
Comment (nikomatsakis) [inline src/hir.md]: Nit: `cargo rustc -- -Zunpretty=hir-tree` would be easier in any case.

**Comment by nikomatsakis** [comment]
Good idea to mention it.

File: src/hir.md (hunk 1 above)
Comment (theotherphil) [reply src/hir.md]: Ah, I didn't realise you could do this.

Comment (theotherphil) [reply src/hir.md]: Ah, I didn't realise you could do this.

**Pull request merged**